pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1027,10 +1027,16 @@ self: super: {
   # https://github.com/mgajda/json-autotype/issues/25
   json-autotype = dontCheck super.json-autotype;
 
-  # Requires pg_ctl command during tests
-  beam-postgres = overrideCabal super.beam-postgres (drv: {
+  # Aeson dependency version bound not satifised but it builds fine without it
+  beam-core = doJailbreak super.beam-core;
+
+  # Aeson dependency version bound not satifised but it builds fine without it
+  beam-migrate = doJailbreak super.beam-migrate;
+
+  # dontcheck: Even though pg_ctl command is provided during tests it doesn't work.
+  beam-postgres = doJailbreak (dontCheck (overrideCabal super.beam-postgres (drv: {
     testToolDepends = (drv.testToolDepends or []) ++ [pkgs.postgresql];
```

I think it's time to simplify this :) How about:

```suggestion
  beam-postgres = overrideCabal super.beam-postgres (drv: {
    jailbreak = true;
    doCheck = false;
    testToolDepends = (drv.testToolDepends or []) ++ [pkgs.postgresql];
```